src/util/http/websocket/QueryToSocketDistributor.h

```diff
@@ -47,6 +47,8 @@ class QueryToSocketDistributor {
   /// update to the data.
   net::awaitable<void> waitForUpdate() const;
 
+  auto useStrandedAwaitable() const;
+
```

Comment:
Should have a very short documentation,  but first look at my proposed changes fro the semantics above.